src/model_signing/api.py

```diff
@@ -32,8 +32,8 @@
 from model_signing.serialization import serialize_by_file
 from model_signing.serialization import serialize_by_file_shard
 from model_signing.signing import in_toto
+from model_signing.signing import sign_sigstore as sigstore
 from model_signing.signing import signing
-from model_signing.signing import sigstore
```

Comment:
Does the `from model_signing.signing` prefix avoid the confusion? 

To clarify, is this to help library users to avoid the confusion?

Reply:
Ack. The recursive import wasn't clear from the diff or PR description, but I understand now.